layouts/css/layout/_sticky-footer.styl

```diff
@@ -9,6 +9,8 @@ body
     flex-direction column
     min-height 100vh
     margin 0
+    overflow-y scroll; /* has to be scroll, not auto */
+    -webkit-overflow-scrolling touch
```

Comment:
I'm not saying your doing wrong. I'm saying its not standard feature. https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling